lib/node.js

```diff
@@ -118,6 +118,7 @@ declare class Buffer extends Uint8Array {
 
 declare module "buffer" {
   declare var kMaxLength: number;
+  declare export var Buffer: typeof Buffer;
```

Comment:
Although without testing I don't know whether the type will refer to the `Buffer` class in the outer scope or just create a vacuous reference to the `var` just declared. cc @samwgoldman